activerecord/test/cases/helper.rb

```diff
@@ -26,6 +26,9 @@
 # Connect to the database
 ARTest.connect
 
+# If QueryComments are enabled, use prepended comments to prevent truncation in setup queries
+ActiveRecord::Base.connection.prepend_comment = true
```

Comment:
This can probably be removed now - I had set it when testing the full suite with query comments enabled.

Reply:
Yea this should be removed. 👍🏻